src/helpers.js

```diff
@@ -112,7 +112,7 @@ function examplesForResource(resource) {
 
 function curlExamplesForAction(action, resource_examples) {
   var action_examples = _.filter(resource_examples, function(example) {
-    var regex = action.path.replace("%s", "\.+").replace(/\//, "\\/");
+    var regex = action.path.replace(/%s/g, "\\.+").replace(/\//g, "\\/");
```

Comment:
Which part doesn't work? I wasn't really sure what the heck was going on with this but as written the path will only have its first instances of `%s` and `/` replaced, and I assumed that we wanted to replace all of them.